server/odc-service/src/main/java/com/oceanbase/odc/config/BeanConfiguration.java

```diff
@@ -67,9 +74,13 @@ public PasswordEncoder passwordEncoder() {
     public ObjectMapper objectMapper(SensitivePropertyHandler sensitivePropertyHandler) {
         CustomOutputSerializer customOutputSerializer = new CustomOutputSerializer()
                 .addSerializer(Internationalizable.class, new I18nOutputSerializer());
+        DialectTypeOutputSerializer dialectTypeOutputSerializer = new DialectTypeOutputSerializer();
+        SimpleModule dialectTypeModule =
```

Comment:
`SimpleModule dialectTypeModule = new SimpleModule().addSerializer(DialectType.class, new DialectTypeOutputSerializer());` may be better

Reply:
done